xlators/mgmt/glusterd/src/glusterd-pmap.c

```diff
@@ -467,16 +435,22 @@ gluster_pmap_portbybrick(rpcsvc_request_t *req)
     return glusterd_big_locked_handler(req, __gluster_pmap_portbybrick);
 }
 
+/* __gluster_pmap_brickbyport() is currently not called anywhere. Keeping
+   the code so that it can be used if anytime needed in future.
+   TBD if implemented: Modify pmap_registry_search as well to search for brick
+   using port */
+
 int
 __gluster_pmap_brickbyport(rpcsvc_request_t *req)
 {
-    pmap_brick_by_port_req args = {
+    /*pmap_brick_by_port_req args = {
```

Comment:
?

Reply:
I have explained in the comment above, but instead of removing I thought of keeping it `commented` so that it can be used easily, if at all needed in the future.